src/api.v2/helpers/worker/workSubmit/submitMarkerHeatmapWork.js

```diff
@@ -1,6 +1,24 @@
-const getExtraDependencies = require('./getExtraDependencies');
 const submitWork = require('./submitWork');
+const workerVersions = require('../workerVersions');
 
+const MarkerHeatmap = 'MarkerHeatmap';
```

Comment:
because of this dependencies thing, moving the etag to the api actually does not solve the problem. It will still be coupled to the code in the UI. isn't it? @cosa65